### PR TITLE
Updated "Get started with WebView2 in WinForms apps"

### PR DESCRIPTION
**Pull request by vchapel**
Changed steps and images to match Visual Studio 2019.

**Rendered articles for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/win32?branch=pr-en-us-1526
_fallback url in case needed:_
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/win32?branch=v-vchapel/WebView2-Get-Started-Doc-Test
* Win32
* WinForms
* WinUI 2
* WinUI 3
* WPF

### CODE REVIEW

File: microsoft-edge/webview2/get-started/winforms.md [hunk 1]
```diff
@@ -181,34 +177,29 @@ Add more controls to your Windows forms from the toolbox, and then process windo
...
+
+1.  Press **F5** to build and run the project.
 
-    Make sure the app display is similar to the following image.
+    Make sure the app looks like the following image.
 
-:::image type="complex" source="./media/winforms-app.png" alt-text="app" lightbox="./media/winforms-app.png":::
-   App
-:::image-end:::
+:::image type="content" source="./media/winforms-app.png" alt-text="app" lightbox="./media/winforms-app.png":::
```
[the remark this answers is not in the document]
Comment (vchapel) [reply microsoft-edge/webview2/get-started/winforms.md:189]: Fixed many, not all.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/get-started/win32.md [hunk 2]
```diff
@@ -247,15 +253,15 @@ Now, WebView should always freeze the `Object` object and returns the page docum
...
+The host and the web content can also communicate with each other through the `postMessage` method.  The web content running within a WebView2 control can post to the host through the `window.chrome.webview.postMessage` method, and the message is handled by any registered the `ICoreWebView2WebMessageReceivedEventHandler` event handler on the host.  Similarly, the host can message the web content through `ICoreWebView2::PostWebMessageAsString` or `ICoreWebView2::PostWebMessageAsJSON` method, which is caught by handlers added from `window.chrome.webview.addEventListener` listener.  The communication mechanism allows the web content to use native capabilities by passing messages to ask the host to run native APIs.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/win32.md:256]:
```suggestion
The host and the web content can also communicate with each other through the `postMessage` method.  The web content running within a WebView2 control can post to the host through the `window.chrome.webview.postMessage` method, and the message is handled by any registered the `ICoreWebView2WebMessageReceivedEventHandler` event handler on the host.

Similarly, the host can message the web content through the `ICoreWebView2::PostWebMessageAsString` or `ICoreWebView2::PostWebMessageAsJSON` method, and the message is caught by handlers that are added from the `window.chrome.webview.addEventListener` listener.  This communication mechanism allows the web content to use native capabilities by passing messages to ask the host to run native APIs.
```

Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/win32.md:256]: motivation for the above change: was missing 'the' before "foo method", in several spots.

Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/win32.md:256]: requires cleanup at "the message is handled by any registered the `ICoreWebView2WebMessageReceivedEventHandler` event handler on the host."  garbled.

**Review by jasonstephen15 — APPROVED** [review]

**Pull request merged**